scripts/check_regression.py

```
@@ -0,0 +1,224 @@
+ベンチマーク結果の回帰検知スクリプト
+
+このスクリプトはCriterionの出力を解析して、性能回帰を検知します。
+CI/CDパイプラインで使用されることを想定しています。
+"""
+
+import json
+import sys
+import os
+import argparse
+from pathlib import Path
+from typing import Dict, List, Optional, Tuple
```

Ruff (0.12.2)

`typing.Tuple` imported but unused

Remove unused import: `typing.Tuple`

(F401)